csl-variables.rnc

```diff
@@ -15,6 +15,7 @@ div {
   ## Date variables
   variables.dates =
     "accessed"
+    | "available-date"
     | "container"
```

Comment:
Not related to you commit: But why is container a date variable?

Reply:
I vaguely remember some issue mentioning this as a bug, but not 100% sure.

Reply:
I don't recall any previous discussion on this, but as far as I can tell, "container" being a date variable dates back to at least 0.8: https://github.com/citation-style-language/schema/blob/v0.8/csl.rnc#L409 (and I was never able to figure out what "container" was for, hence the "?" at https://docs.citationstyles.org/en/1.0.1/specification.html#date-variables)

Reply:
There is only one style in the repository that uses `container` (http://www.zotero.org/styles/wiley-vch-books), and there it just includes it in a list with all of the other date variables. I suggest we remove `container` in this release (https://github.com/citation-style-language/csl-evolution/issues/12). 

If a change to a hierarchical data model ever happens, we can add `container` back in at that time.

Reply:
Removing `container` is a good idea, but what does that mean for our versioning discussing. Then we *theoretically* backwards incompatible, albeit without any real consequences...